.gitignore

```diff
@@ -5,4 +5,3 @@
 
 # infra binary
 dist
-docs/api/openapi3.json
```

Comment:
We still want this line in `.gitignore` because `make docs` can create this file (for release).